js/apps/admin-ui/src/client-scopes/details/ScopeForm.tsx

```diff
@@ -66,7 +77,19 @@ export const ScopeForm = ({ clientScope, save }: ScopeFormProps) => {
 
   useEffect(() => {
     convertToFormValues(clientScope ?? {}, setValue);
-  }, [clientScope]);
+  }, [clientScope, setValue]);
+
+  const handleProtocolSelect = useCallback(
+    (
+      value: string | string[],
+      onChangeHandler: (value: string | string[]) => void,
+    ) => {
+      const protocolValue = Array.isArray(value) ? value[0] : value;
+      onChangeHandler(protocolValue);
+    },
+    [],
+  );
```

Comment:
What is the purpose of this code? As far as I can tell, this will already use the `SingleSelectControl` variant under the hood, so there should be no need to further normalize this value.